common/lib/dependabot/logger.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "logger"
+
+module Dependabot
+  def self.logger
+    @logger ||= Logger.new(STDOUT)
```

Comment:
@jurre thoughts on setting this to nil and the configuring in the updater? Not entirely sure about the best practice here.  Would you add a method like `register_logger`?

Reply:
What about:

```ruby
def self.logger=(logger)
  @logger = logger
end
```